Update: `OpenStreetMap layer` (#1104)
Esri/arcgis-runtime-samples-ios: arcgis-ios-sdk-samples/Layers/OpenStreetMap layer/README.md

## How to use the sample

> The OpenStreetMap basemap will display upon launch. Pan and zoom to explore the basemap.

Reviewer: ```suggestion
The OpenStreetMap basemap will display upon launch. Pan and zoom to explore the base layer.
```

> When the sample opens, it will automatically display the OpenStreetMap basemap. Pan and zoom to explore the basemap.

Reviewer 2: Just one suggestion to update the wording.
```suggestion
The OpenStreetMap basemap will display upon launch. Pan and zoom to explore the basemap.
```

## How it works

> 1. Create an `AGSOpenStreetMapLayer`, and set it as the `AGSMap`'s basemap.
> 2. Set the `AGSMap` to the `AGSMapView`.

Reviewer: The layer is used as the base layer, not the base map. Also, it isn't set on the map; it is provided as initialization (creation) time.

```suggestion
1. Create an instance of `AGSOpenStreetMapLayer`.
2. Create an instance of `AGSMap` using the OpenStreetMap layer as the base layer.
2. Set the map to the map view's `map` property.
```
